java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -1,14 +1,33 @@
 package glide.api.commands;
 
 import glide.api.models.commands.SetOptions;
+import java.util.HashMap;
 import java.util.concurrent.CompletableFuture;
 
-/** String Commands interface to handle single commands that return Strings. */
+/**
+ * String Commands interface.
```

Comment:
Could be a bit confusing, becase string commands don't return strings actually.
```suggestion
 * Commands that perform string manipulation.
```